FrameworkSystem/Client/SystemAdministratorClientCLI.py

```diff
@@ -419,7 +419,8 @@ def getInstallations( self, argss ):
           key = 'UnInstallationTime.smaller'
         elif arg == '-uta':
           key = 'UnInstallationTime.bigger'
-      else:
+      elif isinstance(key, str):
```

Comment:
Fixed str->basestring. The only reason for casting to str is because 
otherwise pylint thinks it's None and trigger an error